.circleci/config.yml

```diff
@@ -31,6 +31,7 @@ commands:
             echo 'export IMAGE_NAME=myapp' >> $BASH_ENV
             echo 'export LIBRA_DUMP_LOGS=1' >> $BASH_ENV
             echo 'export CARGO_INCREMENTAL=0' >> $BASH_ENV
+            echo 'export CI_TIMEOUT="timeout 20m' >> $BASH_ENV
```

Comment:
Unbalanced double quotes.

Reply:
While you're at it, can you force a `-v`?
I fear failing builds without kill signal in the logs are going to be super confusing.

Reply:
It looks like CI uses an old version of `timeout` which doesn't have -v/--verbose option.  It failed the check here https://app.circleci.com/jobs/github/libra/libra/8780
```
timeout: unrecognized option '--verbose'
Try 'timeout --help' for more information.
Exited with code 125
CircleCI received exit code 125
```